src/meta/processors/BaseProcessor.h

```diff
@@ -226,12 +233,21 @@ class BaseProcessor {
     /**
      * Fetch the latest version edge's fields.
      */
-    StatusOr<std::unordered_map<std::string, nebula::cpp2::ValueType>>
-    getLatestEdgeFields(GraphSpaceID spaceId, const std::string& name);
+    std::unordered_map<std::string, nebula::cpp2::ValueType>
+    getLatestEdgeFields(const nebula::cpp2::Schema& latestEdgeSchema);
+
 
-    // Get Edge schema by EdgeType
-    StatusOr<std::unordered_map<std::string, nebula::cpp2::ValueType>>
-    getLatestEdgeFields(GraphSpaceID spaceId, EdgeType edgeType);
+    /**
+     * Fetch the latest version edge's schema.
+     */
+    StatusOr<nebula::cpp2::Schema>
+    getLatestEdgeSchema(GraphSpaceID spaceId, const EdgeType edgeType);
+
+    /**
+     * Check if edge has ttl
+     */
+    StatusOr<bool>
+    edgeHasTTL(GraphSpaceID spaceId, const EdgeType edgeType);
```

Comment:
Do we need the method?

Reply:
This function is deprecated. I remember it has been removed.